packages/rspack/src/config/normalization.ts

```diff
@@ -56,7 +56,11 @@ export const getNormalizedRspackOptions = (
 							name: libraryAsName
 					  } as LibraryOptions)
 					: undefined;
-
+			if (output.filename && output.filename.includes("[ext]")) {
+				throw new Error(
```

Comment:
What if user just wants to use [ext] in their filename

Reply:
"[ext]-[name].js" -> "[ext]-main.js" for example 🤷‍♂️

Reply:
I don't know, i thought about this but I can't find better way

Reply:
or maybe we can keep the alert until we hit 1.0.0 version